src/authorization/entity/user.entity.ts

```diff
@@ -38,6 +38,12 @@ class User extends BaseEntity {
 
   @Column({ nullable: true, default: false })
   public twoFAEnabled?: boolean;
+
+  @Column({ default: 'inactive' })
+  public status!: string;
```

Comment:
Can we make use of the [Typeorm Enum](https://github.com/typeorm/typeorm/blob/master/docs/entities.md#enum-column-type) datatype here?